build.gradle

```diff
@@ -65,8 +66,9 @@ dependencies {
     compile 'org.apache.commons:commons-compress:1.19'
     compile 'org.apache.commons:commons-exec:1.3'
     compile 'org.apache.commons:commons-lang3:3.9'
-    testCompile 'org.mockito:mockito-core:2.23.4'
+    testCompile 'org.mockito:mockito-core:2.28.2'
     testCompile 'org.easytesting:fest-assert:1.4'
+    testCompile 'com.redhat.devtools.intellij:intellij-common:0.0.1-SNAPSHOT:test'
```

Comment:
locally, i cannot find the required intellij-common SNAPSHOT during the compileTestJava phase. 
I suspect that a snapshot repository has to be declared in the gradle file.

Reply:
A new repo has been added

Reply:
ok, seems to be a IDEA setup issues, i had to update the project SDK to use the correct target ( 2020.3 and java11 ) to be able to run the tests.